AI/Nullkiller/Behaviors/Behavior.cpp

```diff
@@ -0,0 +1,5 @@
+namespace Nullkiller
+{
+
+
+}
```

Comment:
interesting file :)

Reply:
Ha-ha. There was some idea) I think it is also not under cmakelist.txt

Reply:
Removed